Packs/CybleEventsV2/ReleaseNotes/1_1_2.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+
+##### CybleEvents v2
+- Improved the fetching logic
+- Added new command called update_alert_data to update status and severity of alerts on Cyble Vision Platform
```

Comment:
```suggestion
- Added new command ***cyble-vision-update_alert_data*** to update status and severity of alerts on Cyble Vision Platform
```
and of course change the name of the command to how you choose to fix it in the code and yml files